scripts/percona-release.sh

```diff
@@ -314,7 +314,9 @@ function enable_alias {
 }
 #
 function check_setup_command {
-  if [[ -n ${2} ]]; then
+  if [[ "$1" == "-y" || "${!#}" == "-y" ]]; then
```

Comment:
you should also show `-y` in help and describe what is it.

Reply:
I've added it to `show_help` function with some examples & explanation.

Also updated check_setup_command